osf/management/commands/export_user_account.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+import io
+import os
+import json
+import logging
+import requests
+import shutil
+
+from django.core import serializers
+from django.core.management.base import BaseCommand
+
+from addons.wiki.models import NodeWikiPage
+from addons.osfstorage.models import OsfStorageFileNode
+from framework.auth.core import Auth
+from osf.models import (
+    OSFUser,
+    PreprintService,
+    Registration,
+)
+from scripts.utils import Progress
+from website.util import waterbutler_api_url_for
+
+PREPRINT_EXPORT_FIELDS = [
+    'is_published',
+    'date_created',
+    'date_modified',
+    'date_published'
+]
+
+NODE_EXPORT_FIELDS = [
+    'title',
+    'is_fork',
+    'category',
+    'is_public',
+    'description',
+    'forked_date',
+    'date_created',
+    'date_modified'
+]
+
+REGISTRATION_EXPORT_FIELDS = NODE_EXPORT_FIELDS + [
+    'registered_data',
+    'registered_meta'
+]
+
+logging.getLogger('urllib3').setLevel(logging.WARNING)
+logger = logging.getLogger(__name__)
+
+ERRORS = []
+
+def export_metadata(node, current_dir):
+    """
+    Exports the pretty printed serialization of a given model instance to metadata.json.
+    Only simple fields (non-FK, non-M2M, etc) are serialized.
+
+    """
+    export_fields = NODE_EXPORT_FIELDS
+    if isinstance(node, Registration):
+        export_fields = REGISTRATION_EXPORT_FIELDS
+    elif isinstance(node, PreprintService):
+        export_fields = PREPRINT_EXPORT_FIELDS
+    with open('{}metadata.json'.format(current_dir), 'w') as f:
+        # only write the fields dict, throw away pk and model_name
+        metadata = serializers.serialize('json', [node], fields=export_fields)
+        f.write(json.dumps(metadata, indent=4, sort_keys=True))
```

Comment:
[Because `serialize()` takes a QS.](https://docs.djangoproject.com/en/1.11/topics/serialization/#serializing-data)

I could've used Django's `model_to_dict()` here, but when I was messing around in the shell the output from it was incomplete (missing `date_created` and `date_modified`). I didn't look much further into it though.

```
In [15]: serializers.serialize('json', [node], fields=NODE_EXPORT_FIELDS)
Out[15]: u'[{"pk": 2, "model": "osf.abstractnode", "fields": {"category": "project", "date_created": "2017-06-28T17:24:27.271Z", "date_modified": "2017-06-28T18:47:31.003Z", "description": "", "forked_date": null, "is_fork": false, "is_public": false, "title": "redirect link testing"}}]'

In [16]: model_to_dict(node, fields=NODE_EXPORT_FIELDS)
Out[16]:
{'category': u'project',
 'description': u'',
 'forked_date': None,
 'is_fork': False,
 'is_public': False,
 'title': u'redirect link testing'}
```